tests/fixtures/hypothesis_configuration.py

```diff
@@ -12,6 +12,7 @@
 settings.load_profile("development")
 
 st.register_type_strategy(str, st.text(min_size=1))
+st.register_type_strategy(float, st.floats(min_value=0.0))
```

Comment:
```suggestion
```
Aren't the tests passing even with negative `float` values?
Alternatively, if a field should be a positive float, maybe we could use pydantic's `NonNegativeFloat` type? (this should constrain hypothesis to generate only positive values)

Reply:
In this case, I change the `float` type for field in this PR as `NonNegativeFloat`